### PR TITLE
fix: load VITE_BASE_CLIENT_URL from .env

### PR DESCRIPTION
**Pull request by Bushuo**
## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Review by absurd-turtle — APPROVED** [review]

File: frontend/.env.sample [hunk 1]
```diff
@@ -2,3 +2,4 @@
 // More information about env variables with Vite can be found here: https://vitejs.dev/guide/env-and-mode.html
 
 VITE_BASE_API_URL=http://localhost:8080
+VITE_BASE_CLIENT_URL=http://localhost:5173
```
Comment (markus2330) [inline frontend/.env.sample:5]: Please write a bit what BASE_CLIENT_URL is and how to use is (in frontend's README.md)

File: frontend/vite.config.ts [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
+
+  return {
+    base: env.VITE_BASE_CLIENT_URL,
+    plugins: [
+      react(),
+      svgr(),
+      VitePWA({
+        manifest,
+        includeAssets: ['favicon.svg', 'favicon.ico', 'robots.txt', 'apple-touch-icon.png'],
+        // switch to "true" to enable sw on development
```
Comment (markus2330) [inline frontend/vite.config.ts:24]: What is "sw"?

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, thanks for the quick fix. Please add a bit more documentation, then it is fine!

File: frontend/.env.sample (hunk 1 above)
Comment (Bushuo) [reply frontend/.env.sample:5]: Yes, about that.
I don't know why exactly this was introduced. I have to ask @badnames about that.

Maybe we can just remove the environment variable overall, as I think the default of '/' is fine in our setup.
Vite will append the variable as base path for static files it tries to serve.
https://vitejs.dev/guide/build.html#public-base-path

Comment (markus2330) [reply frontend/.env.sample:5]: Yes, if it is possible, removing the env var would be best!

@badnames can you maybe shine light on it?

Comment (Bushuo) [reply frontend/.env.sample:5]: I will remove it bcs he added it by mistake.

File: frontend/vite.config.ts (hunk 2 above)
Comment (Bushuo) [reply frontend/vite.config.ts]: I added a comment about that.

**Review by markus2330 — APPROVED** [review]
Thanks for keeping it simple!

**Pull request merged**